src/Core/Components/DateTime/FluentCalendar.razor

```diff
@@ -43,7 +43,7 @@
             @foreach (var weekDay in CalendarExtended.GetDayNames())
             {
                 <div class="week-day" part="week-day" title="@weekDay.Name" abbr="@weekDay.Name">
-                    @weekDay.Abbreviated[0]
+                    @weekDay.Abbreviated
```

Comment:
Revert this

Reply:
The original approach here was to take the first character of the string, which I think is also incorrect

Reply:
No it is noy incorrect. We get the correct abbreviation in the `GetDayName` method and then here we decide to only use the first letter because of space restrictions.